MAINTAINERS.md

```diff
@@ -159,3 +157,5 @@ An automated list can be found [here](https://github.com/orgs/tldr-pages/people)
   until [18 January 2018](https://github.com/tldr-pages/tldr/issues/1878#issuecomment-358610454)
 - Waldir Pimenta ([@waldyrious](https://github.com/waldyrious)):
   until [26 August 2018](https://github.com/tldr-pages/tldr/issues/2257)
+- Zlatan Vasović ([@zlatanvasovic](https://github.com/zlatanvasovic)):
+  [18 June 2020](https://github.com/tldr-pages/tldr/issues/4113) — [14 December 2021](https://github.com/tldr-pages/tldr/issues/7538)
```

Comment:
Looks like this is the correct format
```suggestion
  until [14 December 2021](https://github.com/tldr-pages/tldr/issues/7538)
```

Reply:
I believe that was used for original organization owners, before the introduction of the maintainers system. This way there is no information when did I become an organization owner, but I'm fine with both formats.

Reply:
I think @navarroaxel's suggestion is right. But I don't understand what @zlatanvasovic is saying fully.

Reply:
It's fine, you can accept his suggestion. :smile: